src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet= { tweetsArray[0] } />
```

Comment:
In order to render each tweet and doesnt matter the length of the array we can use the map function, also keep in mind that we we use a map function react expect to add a key prop to the component in order to identify which component you are refering, in general we have and id when we are rendering data. in this case we dont have an id so we can do something in this case like this is use the name of the user ONLY for this case

```suggestion
       {tweetsArray.map((tweet) => {
   return <Tweet tweet = {tweet} key={tweet.user.name}/>;
      })}
```